### PR TITLE
[hi] Add content/hi/docs/tutorials/kubernetes-basics/expose/_index.md

### PR DESCRIPTION
**Pull request by rajatgupta24**
Hind localization of the file: `content/hi/docs/tutorials/kubernetes-basics/expose/_index.md`

PR is linked under this issue #29353

/language hi

### CODE REVIEW

File: content/hi/docs/tutorials/kubernetes-basics/expose/_index.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: अपने ऐप को सार्वजनिक रूप से एक्सपोज़ करें
```
Comment (mittalyashu) [inline content/hi/docs/tutorials/kubernetes-basics/expose/_index.md:2]: Can you use `Publishing the application` to localised in Hindi?

I checked the docs for JP and they using `アプリケーションの公開` means application publishing.

**Comment by anubha-v-ardhan** [comment]
/retitle [hi] Add content/hi/docs/tutorials/kubernetes-basics/expose/_index.md

**Comment by anubha-v-ardhan** [comment]
@rajatgupta24 Can you change the base branch to `dev-1.22-hi.1` and squash the commits into one commit.
That should make this PR ready to merge.

**Review by anubha-v-ardhan — COMMENTED** [review]
@rajatgupta24 Can you change the base branch to `dev-1.22-hi.1`. This PR is ready to merge.

**Comment by anubha-v-ardhan** [comment]
Thank you @rajatgupta24 :)

**Pull request merged**